frontends/web/src/routes/account/send/send.tsx

```diff
@@ -677,7 +677,7 @@ class Send extends Component<Props, State> {
                         onInput={this.handleFiatInput}
                         disabled={sendAll}
                         error={amountError}
-                        value={fiatAmount}
+                        value={fiatAmount.replace('\'','')} /* input with type=number doesn't like some formatting chars like [']. */
```

Comment:
this is a bit hacky and not so robust, i.e. likely to break (e.g. if we add different locales formatting).

why not deliver the fiat amount in the same format as before this PR from the backend?

Reply:
I don't like that too, but we decided to move and centralize all the formatting in the backend, and `FormatAsCurrency` adds apostrophes https://github.com/Beerosagos/bitbox-wallet-app/blob/formatting-refactor/backend/coins/coin/conversions.go#L21
This clearly has to be changed to allow locales support, just like `FormatAsCurrency` in backend. Maybe is it better to change both in a dedicated PR for locales?

Reply:
Sure, but who says that `getConvertToFiatHandler()` needs to format the amount with a locale? it can just return a plain string that is suitable for this input, e.g. instead of `coinpkg.FormatAsCurrency(convertedAmount, to)` use `convertedAmount.FloatString(2)` (or 8 in case of BTC)?

Reply:
Fixed, I made a dedicated `FormatAsPlainCurrency` for this